cookbook/form/form_customization.rst

```diff
@@ -160,7 +160,7 @@ As you can see, this fragment itself renders another fragment - ``form_widget_si
 
     .. code-block:: html+jinja
 
-        {# form_div_layout.html.twig #}
+        {# form_widget_simple.html.twig #}
```

Comment:
`form_div_layout.html.twig` is actually right. The Twig form theme is not split in different files per widget.